plugins/generator-1.18.2/forge-1.18.2/procedures/entity_is_in_powder_snow.java.ftl

```diff
@@ -0,0 +1 @@
+(${input$entity}.isInPowderSnow())
```

Comment:
typo: isInPowderSnow is not a method but rather just a variable, this will fix the build problems